code/src/components/Name.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export const Name = ({ name, setName, counter }) => {
+  const handleNameChange = (event) => {
+    setName(event.target.value);
+
+    console.log('Name:', event.target.value);
+  }
+
+  return (
+    <div className="name-container">
+      <p>What is your name?</p>
+      <form>
+        <section className="inputName">
+          <input className="inputClass" type="text" onChange={handleNameChange} value={name} counter={counter} aria-labelledby="name-label" arial-label="Write your name here and then press button further down" />
```

Comment:
Great use of aria-labels. Think about the className for this input, and if it could be re-named to something more unique to this specific input/component